contents/docs/api/events.mdx

```diff
@@ -6,154 +6,417 @@ showTitle: true
+                        "$screen_height": 1050,
+                        "$screen_width": 1680,
+                        "$lib": "web",
+                        "$lib_version": "1.9.7",
+                        "$insert_id": "esc00tyybcvdp7d4",
+                        "$time": 1619773018.936,
+                        "distinct_id": "1791c3860c9fb1-01d1c764f567fc8-48183301-1aeaa0-1791c3860ca116a",
+                        "$device_id": "1791c3860c9fb1-01d1c764f567fc8-48183301-1aeaa0-1791c3860ca116a",
+                        "$initial_referrer": "http://localhost:1313/",
+                        "$initial_referring_domain": "localhost:1313",
+                        "$referrer": "http://localhost:1313/portfolio/",
+                        "$referring_domain": "localhost:1313",
+                        "$active_feature_flags": [],
+                        "token": "xF2IWfyNR7gX6NeBi-kXBjaYUQw_gcZkQ64PB3RZdlY",
+                        "$ip": "49.207.221.188"
                     },
-                    "order": 1
+                    "event": "$pageleave",
+                    "timestamp": "2021-04-30T08:56:59.450000+00:00",
+                    "person": null,
+                    "elements": [],
+                    "elements_chain": ""
                 },
+            ],
+            "properties": {},
+            "matching_events": [],
+            "email": null,
+            "session_recordings": [
                 {
-                    "event": null,
-                    "text": null,
-                    "tag_name": "body",
-                    "attr_class": null,
-                    "href": null,
-                    "attr_id": null,
-                    "nth_child": 2,
-                    "nth_of_type": 1,
-                    "attributes": {},
-                    "order": 2
+                    "id": "17921fdada3f5-02688ca8f7c53b8-48183301-1aeaa0-17921fdadb72a2",
+                    "viewed": false
                 }
-            ],
-            "event": "$autocapture",
-            "timestamp": "2020-05-19T14:26:31.128297Z",
-            "person": "tim@posthog.com"
+            ]
         }
-    ]
+    ],
+    "pagination": null
 }
 ```
+
+</HiddenSection>
+
+<Endpoint endpoint='/api/event/session_recording' allowedMethods={['get']} />
+
+Get a list session recordings for the events from PostHog.
+
+<HiddenSection headingType='h4' title='Query Parameters'>
+
+
+| Parameter | Type / Valid Values | Description | Required |
+| --- | --- | --- | --- |
+| `session_recording_id` | `string` | ID of the session recording for which to request the events on. | ❌ |
+| `save_view` | `boolean` | Wheether to mark this session recording as "viewed". | ❌ |
+
+</HiddenSection>
+
+<HiddenSection headingType='h4' title='Example'>
+
+##### Request
+
+```
+GET /api/event/sessions/?session_recording_id=179221e51743cd-081d4a8b3f0596-103c6054-1aeaa0-179221e51758ab
+```
+
+##### Response
+
+```json
+HTTP 200 OK
+Content-Type: application/json
+
```

Comment:
Thank you so much for this! It's awesome. I made some changes and filled in some gaps. The only main thing missing is this response

Reply:
Thank you!!! Just added the response with the first event as discussed on slack.

Reply:
Awesome, thanks!